packages/terra-graphs-docs/src/terra-dev-site/doc/Graphs/ScatterGraph/Reflow/DynamicData.2.doc.mdx

```diff
@@ -0,0 +1,39 @@
+import ScatterGraphPanning from '../../../example/Graphs/ScatterGraph/Reflow/DynamicData?dev-site-example';
+import PropsTable from '@cerner/terra-graphs/lib/components/Line/LineGraph?dev-site-props-table';
```

Comment:
Since it is a Scatter graph can you please change the relative path to the scatter component?